src/mudlet.cpp

```diff
@@ -2058,9 +2053,10 @@ void mudlet::slot_show_connection_dialog()
     auto pDlg = new dlgConnectionProfiles(this);
     connect(pDlg, SIGNAL(signal_establish_connection(QString, int)), this, SLOT(slot_connection_dlg_finished(QString, int)));
     pDlg->fillout_form();
-    if (pDlg->exec() == QDialog::Accepted) {
```

Comment:
Note: `QDialog::exec()` is _blocking_ - execution does not return from it until the user closed the dialog - and by default it is `application modal` i.e. the user cannot interact with any other of the application's windows until that happens.  For the ___first___ profile being opened that is probably what is wanted - but may not be good news if multi-playing is in progress.

On the other hand `QDialog::show()` will return after it has made the `QDialog` visible but I am not sure that our code will operate as before if the execution of `slot_show_connection_dialog()` (which is executed when any of the "connect" actions:
+ the main toolbar "Connect" button is clicked
+ the main menu "Play" item is selected
+ a redundant, not shown, unused (and unwanted) `QAction` mactionConnect were to be activated (but it cannot as it is not attached to a widget)

is activated - then reaches it's end before the dialog code has finished executing?